RATapi/run.py

```diff
@@ -19,23 +61,24 @@ def run(project, controls):
     }
 
     horizontal_line = "\u2500" * 107 + "\n"
-
+    display = controls.display != Display.Off
     problem_definition, cells, limits, priors, cpp_controls = make_input(project, controls)
 
-    if controls.display != Display.Off:
+    if display:
         print("Starting RAT " + horizontal_line)
 
     start = time.time()
-    problem_definition, output_results, bayes_results = RATapi.rat_core.RATMain(
-        problem_definition,
-        cells,
-        limits,
-        cpp_controls,
-        priors,
-    )
+    with ProgressBar(display=controls.display != Display.Off):
```

Comment:
```suggestion
    with ProgressBar(display=display):
```
as you've already defined `display = controls.display != Display.Off` in line 64

Reply:
Good catch, I missed this line